package.json

```diff
@@ -0,0 +1,46 @@
+{
+  "name": "eslint-config-web3studio",
+  "version": "1.0.0",
+  "description": "Web3Studio JS config",
```

Comment:
Same as a previous comment - but why does eslint have to be in the name? It's a little confusing that it isn't consistent with the description

Reply:
Agreed, I tried it originally as `web3studio-js-standard` but eslint wouldn't take it. 

For use natively as `extends` you can either be `eslint-config-*` or reference as a file. 

Docs: https://eslint.org/docs/developer-guide/shareable-configs#creating-a-shareable-config

We _could_ do 

```json
{
  extends: './node_modules/web3studio-js-standards/eslint.js'
```

But that seemed worse to me. I could also publish the prettier file separately, but sense eslint and prettier are being used in tandem that didn't seem right either. Naming it only half correct seemed to be the lesser of the two evils. I'll update this description though.